lib/index.js

```diff
@@ -66,6 +65,7 @@ PG.prototype.connect = function(config, callback) {
   config.idleTimeoutMillis = config.idleTimeoutMillis || config.poolIdleTimeout || defaults.poolIdleTimeout;
   config.log = config.log || config.poolLog || defaults.poolLog;
 
+  var poolName = JSON.stringify(config || {});
```

Comment:
Would it make sense to use something like [json-stable-stringify](https://www.npmjs.com/package/json-stable-stringify) here to ensure that matching configs always resolve to the same key? Otherwise there's no guarantee of the order of the keys in the serialized JSON.

I'd imagine that most people have a single place in their code that initializes their connection pool so it's likely nobody has really run into this, but if you have two separate spots and they specify the keys in a different order, they'd map to a different pool:

```js
const a = {foo: 123, bar: 456}
const b = {bar: 456, foo: 123}
JSON.stringify(a)
// '{"foo":123,"bar":456}'
JSON.stringify(b)
// '{"bar":456,"foo":123}'

const stableStringify = require('json-stable-stringify')
stableStringify(a)
// '{"bar":456,"foo":123}'
stableStringify(b)
// '{"bar":456,"foo":123}'
```